packages/validator/src/services/syncCommitteeDuties.ts

```diff
@@ -183,7 +183,7 @@ export class SyncCommitteeDutiesService {
     for (const epoch of [currentEpoch, nextPeriodEpoch]) {
       // Download the duties and update the duties for the current and next period.
       await this.pollSyncCommitteesForEpoch(epoch, indexArr).catch((e: Error) => {
-        this.logger.error("Failed to download SyncDuties", {epoch}, e);
+        this.logger.warn("Failed to download SyncDuties", {epoch}, e);
```

Comment:
Reference: https://github.com/ChainSafe/lodestar/issues/5359#issuecomment-1529027981
Not sure if this should be an error? We also should suppress it if it's triggered due to `Node is syncing` state.

Reply:
same as https://github.com/ChainSafe/lodestar/pull/5457#discussion_r1186128363 above